README.md

```diff
@@ -2,87 +2,107 @@
+Of course resources that live outside the scope of namespace/project are displayed, too.
+You can switch the current namespace/project with the context menu item "Use Namespace". 
+In OpenShift clusters the tree also lists projects where the tooling behaves in an equivalent manner. 
 
 #### Resource Categories
-Resources are organized in different categories.
+Resources are grouped in different categories such as Namespaces, Nodes, Workloads, Network, Storage, Configuration and Custom Resources.
 
 ![img.png](images/categories.png)
 
 #### Pods
-Pods are marked as running if their icon has a green dot. 
-A red dot, on the contrary, indicates that a pod is either pending, succeeded (terminated), failed or is in an unknown state.
-Pods also unveil their IP address and the number of running out of the total number of containers.
+Pods are shown with their running state. When running their icon holds a green dot. 
+When red, the dot indicates that a pod is either pending, succeeded (terminated), failed or is in an unknown state.
+Pods also unveil their IP address and the number of running containers out of the total number of containers.
 
 ![img.png](images/pod.png)
 
-### Edit Resources
-You can edit any resource that is listed in the resource tree.
-Either double click it or pick "Edit..." in the context menu. A Yaml editor opens up and allows you to edit the resource.
+### Create and Edit Resources
+You can edit any existing resource that is shown in the resource tree.
+Either double click it or pick "Edit..." in the context menu. 
+A Yaml editor opens up and allows you to change the resource. 
+Pushing the editor to the cluster then updates the resource that you were editing.
+You can also open a local yaml/json file and create a new resource on the cluster by pushing the editor to it.
+
+> **Tip**: You're advised to edit copies of your yaml/json files. 
+> This is because the editor keeps your content in sync with the resource that exists on the cluster. 
+> Changes that happen on the cluster are included when pushing or pulling the editor.  
+    
 
 ![editor](images/editor.png)
 
-The editor allows you to push or pull the resource to or from the cluster. 
-The toolbar holds corresponding actions.
+The editor validates your content on behalf of a schema that's matching your kubernetes resource.
+For instance if you're editing a pod, the editor validates it by a kubernetes pod schema and highlights the portions that are infringing.
 
-![editor toolbar](images/editor-toolbar.png)
+![editor schema](images/editor-schema.png)
 
-The editor provides schema validation for the basic kubernetes types.
+#### Push (Save) to Cluster
 
-![editor schema](images/editor-schema.png)
+You can push your editor to the cluster in order to update an existing resource or create a new one.
+The cluster that the editor is saved to is set in the current context (cluster, namespace and user) that's defined in your kube config.
```

Comment:
```suggestion
Once the changes are deployed on the cluster, the extension sets the current context (cluster, namespace and user) pointing to the cluster as defined in the kubeconfig.
```

Reply:
I guess that my initial text was misleading: I read in your text that the editor is setting the current context which isn't true. I was trying to express that the editor was saving to the current context that's set in the kubeconfig.

Reply:
Thanks for clarifying this @adietish. I think the new wording works. 👍🏼

Reply:
```suggestion
The editor is saving the changes to the current context (cluster, namespace and user) that's defined in the kubeconfig
```